src/include/storage/table/table_heap.h

```diff
@@ -108,12 +108,12 @@ class TableHeap {
   /**
    * Return the begin iterator of the table.
    */
-  TableIterator begin(Transaction *txn);
+  TableIterator Begin(Transaction *txn);
```

Comment:
Needs to be lower case for std iteration.

Reply:
It looks like std::iterator was [deprecated](https://stackoverflow.com/questions/37031805/preparation-for-stditerator-being-deprecated/38103394) in C++17, actually. Perhaps it is OK to break this and open an issue to move to whatever standards are being dreamed up right now.

Reply:
No strong opinion.

Reply:
I'm just going to merge it for now.